test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -436,7 +436,7 @@ public void PackCommand_PackProjectWithPackageType_SnupkgContainsOnlyOnePackageT
             }
         }
 
-        [PlatformTheory(Platform.Windows)]
+        [Theory(Skip = "https://github.com/NuGet/Home/issues/12194")]
```

Comment:
How about using:
[PlatformTheory(Platform.Windows, Skip = "https://github.com/NuGet/Home/issues/12194")]
So that when we tried to re-enable it, we know we will only enable it on Windows？